### PR TITLE
Added WoSign to Security and PKI section

### PR DESCRIPTION
**Pull request by DeniSix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ Table of Contents
   * http://vaddy.net/ - Continuous web security testing with continuous integration (CI) tools. 3 domains, 10 scan history for free
   * https://www.globalsign.com/en/ssl/ssl-open-source/ - Free SSL certs for Open Source projects
   * https://www.startssl.com/ - Free SSL certs
+  * https://buy.wosign.com/free/ - Free SSL certs. Up to 100 subdomains. China authority
```
Comment (ripienaar) [inline README.md]: the cert on their own site is a SHA-1 cert and the CA cert is also sha-1, they seem to suggest they can issue sha-2 certs for the free ones but meh, rather not support people who still use sha-1 anyway

Comment (rekby) [inline README.md]: It disable free 100 domains/subdomains for certificate. 
Now free certificate avalaible for one domain only (+www subdomain - free too), more domains have a small cost.

**Comment by ripienaar** [comment]
they seem to have fixed their own SSL, if @DeniSix  wants to update the product description we can merge it

It says "For Free SSL Certificate, it support 5 domain names for 2 years period"

**Comment by DeniSix** [comment]
Updated. It's not so good now, but still useful :wink:

**Comment by ripienaar** [comment]
:+1:

**Pull request merged**